learning_resources/serializers.py

```diff
@@ -129,17 +163,27 @@ class Meta:
         exclude = ("learning_resource", *COMMON_IGNORED_FIELDS)
 
 
-class MicroRelationshipSerializer(serializers.ModelSerializer):
+class MicroLearningPathRelationshipSerializer(serializers.ModelSerializer):
     """
-    Serializer containing only the parent and child ids
+    Serializer containing only parent and child ids for a learning path relationship
     """
 
     class Meta:
         model = models.LearningResourceRelationship
         fields = ("id", "parent_id", "child_id")
 
 
-class LearningResourceBaseSerializer(serializers.ModelSerializer):
+class MicroUserListRelationshipSerializer(serializers.ModelSerializer):
```

Comment:
Do you mind returning these as `parent/child`:

```
    parent = serializers.ReadOnlyField(source="parent_id")
    child = serializers.ReadOnlyField(source="child_id")
```

#77 Does this also `MicroLearningPathRelationshipSerializer` ... Maybe just change them both here and I can deal with the merge conflict? Since I'm out next week I expect this PR will be merged before #77 

**Rationale:** That way responses from MicroUserListRelationshipSerializer & UserListRelationshipSerializer are more similar.